app/src/main/groovy/io/seqera/events/application/handlers/EventHandler.groovy

```
@@ -64,14 +64,14 @@ class EventHandler extends JsonHandler {
         }
 
         Long pageNumber = queryParamValidator.validatePageNumber(queryParams)
-        if (!pageNumber) {
+        if (!pageNumber || pageNumber < 1) {
             println queryParamValidator.invalidParamsMessageOrDefault("pageNumber")
             sendErrorResponse(http, queryParamValidator.invalidParamsMessageOrDefault("pageNumber"), HttpStatus.BadRequest.code)
             return
         }
 
         Integer itemCount = queryParamValidator.validateItemCount(queryParams)
-        if (!itemCount) {
+        if (!itemCount || itemCount < 1) {
```

Just a fail fast check, since this was still protected when we create the PageDetails class

---

👍